README.md

```diff
@@ -4,7 +4,11 @@ Rekór - Greek for “Record”
 
 Rekor's goals are to provide an immutable tamper resistant ledger of metadata generated within a software projects supply chain.  Rekor will enable software maintainers and build systems to record signed metadata to an immutable record. Other parties can then query said metadata to enable them to make informed decisions on trust and non-repudiation of an object's life-cycle, based on signed metadata stored within a tamper proof binary (merkle) tree.
 
-Rekor requires a running instance of the [rekor-server](https://github.com/projectrekor/rekor-server).
+## Build Rekor CLI
+
+From `rekor/cmd/server`
+
+`go build -o rekor`
```

Comment:
you have `rekor` here but then use `rekor-cli` below. We should probably make this readme and `.gitignore` consistent.